Cherry/README.md

```diff
@@ -0,0 +1,9 @@
+# Cherry
+
+## Screenshots
+
+![alt text](https://i.ibb.co/tZk7gTJ/Cherry-Spotify1.png)
+![alt text](https://i.ibb.co/G0F2kj9/Cherry-Spotify2.png)
+
+## More
```

Comment:
The font used by the theme seems to be the not  the default one. Please mention

Reply:
Changed images and included link to font :)